src/digitalmarketplace/components/cookie-settings/cookie-settings.js

```diff
@@ -115,6 +117,30 @@ CookieSettings.prototype.hideWarningMessage = function () {
   }
 }
 
+function hideInlineErrorMessage () {
+  var inlineErrorMessage = document.querySelector('#dm-cookie-settings-error-inline')
+  if (inlineErrorMessage !== null) {
+    inlineErrorMessage.style.display = 'none'
+  }
+
+  var inlineErrorHighlight = document.querySelector('.govuk-form-group--error')
+  inlineErrorHighlight && inlineErrorHighlight.classList.remove('govuk-form-group--error')
+}
+
+function showInlineErrorMessage () {
+  var firstFormGroup = document.querySelector('.govuk-form-group')
+  firstFormGroup && firstFormGroup.classList.add('govuk-form-group--error')
+
+  var inlineErrorSpan = document.createElement('span')
+  inlineErrorSpan.setAttribute('id', 'dm-cookie-settings-error-inline')
+  inlineErrorSpan.className = 'govuk-error-message'
+  inlineErrorSpan.innerHTML = '<span class="govuk-visually-hidden">Error:</span> Please select \'Yes\' or \'No\''
+
+  var parentElement = document.getElementsByClassName('govuk-radios govuk-radios--inline')[0]
+  var grandparentElement = parentElement.parentElement
+  grandparentElement.insertBefore(inlineErrorSpan, parentElement)
```

Comment:
I'm not fully sure where this is going to end up - looking at the current form's HTML, we have

```
<form id="dm-cookie-settings" data-module="dm-cookie-settings">
  <div class="govuk-form-group">
    <fieldset class="govuk-fieldset"> 
      <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">Do you want to accept analytics cookies</legend>
      <div class="govuk-radios govuk-radios--inline">
         ....
      </div>
    </fieldset>
  </div>
</form>
``` 

So am I right that the `inlineErrorSpan` would be inserted before the `<fieldset>`? On the [Design System example](https://design-system.service.gov.uk/components/error-message/) the inline error is right before the element itself, so do we need the parent/grandparent?

Reply:
I think `grandparentElement.insertBefore(inlineErrorSpan, parentElement)` should insert the `inlineErrorSpan` before the `parentElement` (in this case the radios div)?

Reply:
Ahh I see - I think the naming confused me a bit (should `parentElement` be `siblingElement` then?)